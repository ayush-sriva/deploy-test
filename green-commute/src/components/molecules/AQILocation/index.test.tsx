import renderer from 'react-test-renderer';
import AQIIndexGroup from '.';

describe('AQIIndexGroup', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<AQIIndexGroup index='894' location='Delhi' />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
