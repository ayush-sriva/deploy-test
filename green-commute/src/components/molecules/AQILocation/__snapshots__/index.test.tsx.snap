// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AQIIndexGroup renders correctly 1`] = `
<div
  className="MuiGrid-root MuiGrid-container makeStyles-container-1 css-11lq3yg-MuiGrid-root"
>
  <div
    className="MuiGrid-root MuiGrid-item makeStyles-aqi-2 css-13i4rnv-MuiGrid-root"
  >
    <span
      className="MuiTypography-root MuiTypography-body3 makeStyles-innerText-3 css-37jdci-MuiTypography-root"
    >
      894
    </span>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item makeStyles-item-5 css-13i4rnv-MuiGrid-root"
  >
    <h3
      className="MuiTypography-root MuiTypography-h3 makeStyles-text-4 css-gepadz-MuiTypography-root"
    >
      Delhi
    </h3>
  </div>
</div>
`;
